recipes/cgview/meta.yaml

```diff
@@ -28,8 +28,3 @@ requirements:
        - perl
        - perl-bioperl
 
-test:
-    commands:
-        - cgview >/dev/null 2>&1
```

Comment:
Can we stick to this but use the following trick: `cgview 2>&1 | grep "some message in the output"`

Reply:
That would be seem more sensible...